pkgs/development/tools/rust/rustup/default.nix

```
@@ -53,6 +53,8 @@ rustPlatform.buildRustPackage rec {
     done
     popd
 
+    wrapProgram $out/bin/rustup --prefix "LD_LIBRARY_PATH" : "${libPath}"
```

since rustup is an elf, i would prefer to patchelf it instead:
```suggestion
    rpath=$(patchelf --print-rpath $out/bin/rustup)
    patchelf --set-rpath "$rpath:${libPath}" $out/bin/rustup
```

---

Unfortunately this does not solve the issue:
```
$> ./result/bin/rustc             
/home/tobias/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/bin/rustc: error while loading shared libraries: libz.so.1: cannot open shared object file: No such file or directory
```

---

ah, I see, it calls another executable. The current wrapping is fine then :)